imagepipeline-base/src/main/java/com/facebook/imageformat/ImageFormatChecker.java

```diff
@@ -50,19 +50,20 @@ public ImageFormat determineImageFormat(final InputStream is) throws IOException
     final byte[] imageHeaderBytes = new byte[mMaxHeaderLength];
     final int headerSize = readHeaderFromStream(mMaxHeaderLength, is, imageHeaderBytes);
 
+    ImageFormat format = mDefaultFormatChecker.determineFormat(imageHeaderBytes, headerSize);
```

Comment:
Roger that. I will revert these changes and open another pull request that features them.